pkg/volume/secret/secret.go

```diff
@@ -43,7 +43,8 @@ const (
 
 // secretPlugin implements the VolumePlugin interface.
 type secretPlugin struct {
-	host volume.VolumeHost
+	host          volume.VolumeHost
```

Comment:
Should we simply ask the host to provide a `GetSecret` method? This file doesn't need to access the kubeclient at all with the new secret manager.